test/gtest/aca_test_dhcp.cpp

```diff
@@ -16,9 +16,53 @@
 #define private public
 #include "aca_dhcp_server.h"
 #include "aca_dhcp_programming_if.h"
-
+#include "aca_net_config.h"
+#include "aca_ovs_l2_programmer.h"
+#include "aca_comm_mgr.h"
+#include "aca_util.h"
+#include "goalstate.pb.h"
+
+using namespace std;
+using namespace alcor::schema;
 using namespace aca_dhcp_server;
 using namespace aca_dhcp_programming_if;
+using aca_comm_manager::Aca_Comm_Manager;
+using aca_net_config::Aca_Net_Config;
+using aca_ovs_l2_programmer::ACA_OVS_L2_Programmer;
+
+static string project_id = "99d9d709-8478-4b46-9f3f-000000000000";
```

Comment:
since most of these has been defined in aca_test_ovs_l2.cpp, you can simply do the below to reuse them:

// extern the string and helper functions from aca_test_ovs_l2.cpp
extern string project_id;
extern string vpc_id_1;
extern string vpc_id_2;
extern string subnet_id_1;
extern string subnet_id_2;
...